### ISSUE TITLE
Autopreview not working on startup

### ISSUE THREAD

**Issue by ghost** [brrd/Abricotine#182]
On the startup of Abricotine (by opening a file), the checkboxes of task lists will not be displayed as checkboxes. Instead you will see the source code - [ ].
By clicking on any of the task list entries the checkboxes will appear for the whole document.

Linux Mint 18.1 Cinnamon, Abricotine 0.5, own template

![grafik](https://cloud.githubusercontent.com/assets/24965001/25743948/7686a61e-3197-11e7-87a6-cddf90bea674.png)

**Comment by krbaehrle** [comment]
Source code on startup:
```
<pre class=" not-blank CodeMirror-line " role="presentation">
	<span role="presentation" style="padding-right: 0.1px;">
		<span class="cm-m-markdown cm-formatting cm-formatting-list cm-formatting-list-ul cm-variable-2">* </span>
		<span class="cm-m-markdown cm-formatting cm-formatting-task cm-meta">[ ]</span>
		<span class="cm-m-markdown cm-variable-2"> Test</span>
	</span>
</pre>
```

Source code after clicking
```
<pre class=" not-blank CodeMirror-line " role="presentation">
	<span role="presentation" style="padding-right: 0.1px;">
		<span class="CodeMirror-widget" role="presentation" cm-ignore-events="true">
			<span class="autopreview-todolist todolist"></span>
		</span>
		<span class="cm-m-markdown cm-variable-2">Test</span>
	</span>
</pre>
```

**Comment by brrd** [comment]
It seems to be solved in version 0.6.0.
